lib/test.gi

```diff
@@ -329,8 +329,12 @@ InstallGlobalFunction("Test", function(arg)
              if d[1] <> '-' then
                d := Concatenation("+", d);
              fi;
-             Print("########> Time diff in ",
-                   fnam,", line ",line,":\n");
+             Print("########> Time diff in ")
```

Comment:
No semicolon in this line - that's what causing Travis tests to fail. Please fix it and I suggest to merge it - no objections expressed during three weeks.